docs/api.md

```diff
@@ -0,0 +1,40 @@
+Short API doc.
+
+# Authentication
```

Comment:
Since we decided to develop a telegram bot, we need to connect a particular telegram account to the user of a calendar. Does it make sense to have separate methods to grant and revoke access by token?